src/objects/GetStartedChecklistSection__mdt.object

```diff
@@ -39,6 +65,6 @@
         <unique>false</unique>
     </fields>
     <label>NPSP Get Started Checklist Section</label>
-    <pluralLabel>NPSP Get Started Checklist Sections</pluralLabel>
+    <pluralLabel>NPSP Get Started Checklist Section</pluralLabel>
```

Comment:
Did you remove the "s" from the plural label? If not, can you put it back in?
(If you agree @smarovich)

Reply:
Agreed!